src/messages/GE.json

```diff
@@ -356,13 +356,12 @@
   "This response contains an @odata property.": "This response contains an @odata property.",
   "Click here to follow the link": "Click here to follow the link",
   "Signing you out...": "Signing you out...",
-  "As user": "(as user)",
-  "As Teams app": "(as Teams app)",
-  "Use Explorer as sample Teams application": "Use Explorer as sample Teams application",
-  "Use Explorer as logged-in user": "Use Explorer as logged-in user",
-  "Resource Specific Consent popup": "To use application permissions you must use resource specific consent by downloading the ​",
-  "Sample Explorer Teams app": "Sample Explorer Teams app",
-  "Application Permissions": "Application Permissions",
+  "As user": "(signed in as user)",
+  "Use Graph Explorer as a sample Teams application": "Use Graph Explorer as a sample Teams application",
+  "Use Graph Explorer as a signed in user": "Use Graph Explorer as a signed in user",
+  "Resource Specific Consent popup": "To use resource-specific consent permissions, download​",
+  "Sample Teams app for Graph Explorer": "Sample Teams app for Graph Explorer",
+  "Install sample app": "Install sample app",
   "Learn more about": "Learn more about",
   "resource specific consent": "resource specific consent",
```

Comment:
does this need a hyphen between resource and specific as well?

Reply:
Ah, wasn't thinking about internal variables! ad3c65b773d07bd438c8de531f8a0e8161f03bda

Reply:
np! if it's not outside your scope maybe you can do a repo-wide search for "resource consent" to see how widespread this missing hyphenation issue is